clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -50,7 +47,8 @@ class GCEUtil {
 
   // TODO(duftler): This list should not be static, but should also not be built on each call.
   static final List<String> baseImageProjects = ["centos-cloud", "coreos-cloud", "debian-cloud", "google-containers",
-                                                 "opensuse-cloud", "rhel-cloud", "suse-cloud", "ubuntu-os-cloud"]
+                                                 "opensuse-cloud", "rhel-cloud", "suse-cloud", "ubuntu-os-cloud",
```

Comment:
Would it make sense for this list to be a config property?
That way you could "easily" prune the list to things the organization may care about, as well as possibly add (or entirely replace with) other projects exporting sanctioned images for the local organization(s).